DotNetHowToSynonyms/DotNetHowToSynonyms/Program.cs

```diff
@@ -58,13 +56,13 @@ private static SearchServiceClient CreateSearchServiceClient()
         private static SearchIndexClient CreateSearchIndexClient()
         {
             string searchServiceName = ConfigurationManager.AppSettings["SearchServiceName"];
-            string queryApiKey = ConfigurationManager.AppSettings["SearchServiceQueryApiKey"];
+            string queryApiKey = ConfigurationManager.AppSettings["SearchServiceApiKey"];
```

Comment:
Okay

---
In reply to: [111047416](https://github.com/Azure-Samples/search-dotnet-getting-started/pull/10#discussion_r111047416) [](ancestors = 111047416)